docs/06-appearance.md.lua

```diff
@@ -229,8 +70,9 @@ f:write[[
 
 Beautiful is where Awesome theme variables are stored.
 
+<br \><br \>
 ]]
-f:write(create_table(beautiful_vars, {"link", "desc"}))
+f:write(parser.create_table(beautiful_vars, {"link", "desc"}))
```

Comment:
Looks like this only changes the code to use the new `parser` (so should be done in the commit that adds `parser`, I guess)... and then, some new `<br \><br \>` appear that seem totally unrelated. Why?

Reply:
> appear that seem totally unrelated. Why?

It was already there https://github.com/awesomeWM/awesome/blob/master/docs/06-appearance.md.lua#L166